spec/lib/blog_fetcher_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 describe BlogFetcher do
- describe ".feed" do
+  describe ".feed" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.